suite-native/assets/src/components/NetworkAssetsBottomSheet.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { G } from '@mobily/ts-belt';
+
+import { BottomSheet } from '@suite-native/atoms';
+import { AccountsListGroup } from '@suite-native/accounts';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { AccountKey, TokenAddress } from '@suite-common/wallet-types';
+import { selectAccountsByNetworkSymbol } from '@suite-common/wallet-core';
+
+type NetworkAssetsBottomSheetProps = {
+    networkSymbol: NetworkSymbol | null;
+    onSelectAccount: (accountKey: AccountKey, tokenContract?: TokenAddress) => void;
+    onClose: () => void;
+};
+
+export const NetworkAssetsBottomSheet = ({
```

Comment:
What is this file good for? Seems unused.

Reply:
It is used in `Assets.tsx`.